src/openforms/conf/base.py

```diff
@@ -231,6 +231,7 @@
     # Apps registering static variables
     "openforms.variables.static_variables.apps.StaticVariables",
     "openforms.authentication.static_variables.apps.AuthStaticVariables",
+    "openforms.pre_requests",
```

Comment:
we don't need to add it to `INSTALLED_APPS` since:

* there are no models
* there is no `AppConfig.ready()` that imports/loads code